interp/module.go

```diff
@@ -275,21 +275,3 @@ func DefaultOpen(ctx context.Context, path string, flag int, perm os.FileMode) (
 	}
 	return os.OpenFile(path, flag, perm)
 }
-
-func OpenDevImpls(next OpenModule) OpenModule {
```

Comment:
Hey guys, I have some feedback regarding this change. Please let me know of your opinions.

Is there any particular reason why this function was removed? While it's easy to copy it to my projects, I think it's useful to have it here so new users can find and use it. This is a common use case IMO. I noticed that you even had to copy it to the tests files to keep it green.

If you decide to proceed with the removal, it would be nice to at least mention it in the CHANGELOG. IIRC this was here since v2.x.

/cc @kaey @mvdan

Reply:
Have you seen #419? It contains all the context, I think.

I don't have a strong opinion here, to be honest. I initially added it with the idea that more devices would be added with time, such as `/dev/zero`, `/dev/urandom`, `/dev/stdin`, and so on.

If people would find the above useful, I think it's worthwhile keeping or re-adding the feature. But as it was, barely above six lines of code, I don't think there's a strong argument for keeping it there.

Reply:
I do agree that this belongs in the changelog if we decide to remove it, though.

Reply:
Got it.

Another suggestion if you proceed with the removal: along with the CHANGELOG mention, an example on the documentation or README would be helpful. I think that people that needs Windows support will want to know how to support `/dev/null`.

Reply:
This function cannot work generally well for all cases due to differences in windows path handling (forward slash/backward slash/mixed).
For simple cases you can get away with 
```
if path == "/dev/null" && runtime.GOOS == "windows" {
    path = "NUL"
}
```

No need for builtin devnull implementation.

Reply:
I agree with @kaey; let's leave this out for now. I'll amend the changelog.